charts/isaac-app/values.yaml

```diff
@@ -49,6 +49,78 @@ isaac-app-renderer:
     type: ClusterIP
   containerPorts:
     http: 80
+  ingress:
+    enabled: true
+    ingressClassName: nginx
+    hostname: editor-preview.isaac.classis.io
+    path: "/"
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-prod
+    extraTls:
+      - hosts:
+          - editor-preview.isaac.classis.io
```

Comment:
Oh and this ingress is needed since the URL is present for use in the content-editor app.